ServerCore/Helpers/SubmissionEvaluator.cs

```diff
@@ -0,0 +1,356 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+
+namespace ServerCore.Helpers

```

Comment:
Comments at least at the file and class level would be helpful if the idea is to move to this going forward